### DISCUSSION TITLE
Time Zones and Discussions

### DISCUSSION THREAD

**Discussion by gbrail** [mozilla/rhino#1066] [General]
We have a number of people regularly working on Rhino right now. This is actually a pretty huge milestone. But we have never seen or spoken to one another. 

Would anyone be willing to join a regular community video call? (Or audio, if you'd rather not share your video.) There's no need for deep technical discussions or arguments, but it might be nice for some of us to introduce ourselves.

To that end, I'm pretty sure that we cover the world, time-zone wise. I'm in the Pacific time zone in the US. 

If any of you are interested, could you drop a note as to what kinds of time zones make sense for you?

**Comment by p-bakker** [comment]
I'm up for that, was already thinking about raising the exact same question
myself, in light of some of open discussion points for the Rhino roadmap
beyond v1.7.14/15.

I'm in CE(S)T (UTC+2 atm), so (early) morning in north America means late
afternoon/evening here

**Reply by rbri** [reply]
Great idea will join. I'm also at Berlin time zone

**Reply by tonygermano** [reply]
I'm in the Eastern time zone in the US (UTC-4 until we change clocks in a couple weeks)

**Comment by p-bakker** [comment]
Should we try to schedule something now that v1.7.14 has been released?

**Comment by p-bakker** [comment]
> https://github.com/mozilla/rhino/pull/1188#issuecomment-1056042066

Lets try to get this show on the road :-) @gbrail @tuchida @rbri @tonygermano @rPraml @carlosame and anyone else who wants to participate

Proposal: Wednesday March 9th at 7PM UTC / 8pm CEST / 2PM EST

**Reply by tuchida** [reply]
I am sorry, but I am not able to speak English. If it is text, I can read to some extent by machine translation.
